src/LibraryUtilities.ts

```diff
@@ -17,6 +17,7 @@ export class TypeListNode {
     keywords: string = "";
     parameters: string = "";
     description: string = "";
+    isNew: boolean = false;
```

Comment:
Oh the latest commit has some of my code just for testing purpose...I'll push another one later after I resolved some conflicts.